plugins/woocommerce/includes/data-stores/class-wc-product-data-store-cpt.php

```diff
@@ -1236,7 +1236,7 @@ public function create_all_product_variations( $product, $limit = -1, $default_v
 
 			do_action( 'product_variation_linked', $variation_id );
 
-			$count ++;
+			++$count;
```

Comment:
[Same](https://github.com/woocommerce/woocommerce/pull/48448/files#r1639764534) with this line.